### DISCUSSION TITLE
Event on cluster click

### DISCUSSION THREAD

**Discussion by Amaurgit** [NathanAP/vue-google-maps-community-fork#60] [Q&A]
Hey there,

Anyone managed to catch a click event on the cluster marker ? :)

Thanks !

**Comment by andyslack** [comment]
Hi @Amaurgit 

It looks like we don't have an event firing for that use case right now.

The only two events are `['clusteringbegin', 'clusteringend']`

Although I don't actually see the event in the docs for this: 

https://developers.google.com/maps/documentation/javascript/events

Do you know if it's possible?
